packages/block-editor/src/store/undo-ignore.js

```diff
@@ -1,4 +1,4 @@
 // Keep track of the blocks that should not be pushing an additional
 // undo stack when editing the entity.
-// See the implementation of `syncDerivedUpdates` and `useBlockSync`.
+// See the implementation of `useBlockSync`.
 export const undoIgnoreBlocks = new WeakSet();
```

Comment:
I believe this variable isn't needed anymore either?

Reply:
I didn't remove it because it was being used in `useBlockSync`. But maybe we can remove [these lines](https://github.com/WordPress/gutenberg/blob/da98bdb025937d8bb0abc41a187792e75e17731d/packages/block-editor/src/components/provider/use-block-sync.js#L277-L280) as well? You know better than me 😄

Reply:
Yep! They are not used anywhere else and were introduced in the same PR.

Reply:
Perfect! I just removed it in [this commit](https://github.com/WordPress/gutenberg/pull/62229/commits/d76e28872d72fd983b95f539e07f980774755b85). Let's see if tests pass.